cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -277,3 +277,22 @@ def _clear_ftp():
                     dt.total_seconds() >= 0.0,
                     "Negative gap between log files ({}, {})".format(
                         datetimes[i][2], datetimes[i + 1][2]))
+
+
+def test_collect_unconfigured(request, soft_assert):
+    """ Test checking is collect button enable and disable after log depot was configured
+
+    """
+    configure.ServerLogDepot.Credentials.clear()
+    log_credentials = configure.ServerLogDepot.Credentials("smb",
+                                                           "testname",
+                                                           "testhost",
+                                                           username="testusername",
+                                                           password="testpassword")
+    log_credentials.update(validate=False)
+    # check button is enable after adding log depot
+    soft_assert(configure.ServerLogDepot.is_collect_button_disabled() is False)
+    configure.ServerLogDepot.Credentials.clear()
+    request.addfinalizer(configure.ServerLogDepot.Credentials.clear)
+    # check button is disable after removing log depot
+    assert configure.ServerLogDepot.is_collect_button_disabled() is True
```

Comment:
soft_assert(cfme.web_ui.toolbar.is_greyed("Collect", "Collect all logs") is True)